### PR TITLE
Add 'time' cost of the score

### PR DESCRIPTION
**Pull request by Shane-XB-Qian**
adding the time for the score, maybe looking a bit more completion or fun .. :)

### CODE REVIEW

File: autoload/killersheep.vim [hunk 1]
```diff
@@ -98,6 +98,7 @@ endfunc
 func s:IntroFilter(id, key)
   if a:key == 's' || a:key == 'S'
     call s:Clear()
+	let s:time_start = reltime()
```
Comment (mattn) [inline autoload/killersheep.vim]: broken indentation

Comment (Shane-XB-Qian) [reply autoload/killersheep.vim]: oh, yes, i used to use 'tab' .. :) 
i modified it, and some others places had such broken also, thx review.

**Comment by Shane-XB-Qian** [comment]
fixed the indentation and typo. mine and previous existed.

**Comment by Shane-XB-Qian** [comment]
// adding the time cost (with `level n`):
image

**Comment by Shane-XB-Qian** [comment]
r.i.p bram.

**Pull request closed without merging**